src/components/ScholarshipCard.tsx

```diff
@@ -288,6 +289,10 @@ export default function ScholarshipCard({
               label={t('actions.reportIssue')}
             />
           )}
+
+          <Typography variant="subtitle2" align="right" color="text.secondary">
+            last modified: {lastModified?.toLocaleDateString()}
```

Comment:
```suggestion
            Last updated: {lastModified?.toLocaleDateString()}
```

Can you add this to the translations?